docs/how-to/upgrade-wordpress-charm.md

```diff
@@ -0,0 +1,20 @@
+# How to upgrade WordPress Charm
+
+Before updating the Charm you need to backup the database using mysql charm's `create-backup` action.
+
+```
+juju run mysql/leader create-backup
+```
+Additional info can be found about backup on [Charmhub](https://charmhub.io/mysql/docs/h-create-and-list-backups)
```

Comment:
Nitpick: "on [Charmhub](link)" --> "in [the MySQL documentation](link)"
(To be more explicit about where the link takes the reader)